docs/README-DEPLOYMENT.md

```diff
@@ -8,9 +8,9 @@ A [`<semver.version>` directory](./README.md#handsontable-docs-directory-structu
 
 Our server configuration watches for images tagged as [`:latest`](./README-EDITING.md#editing-the-latest-docs-version), and automatically refreshes after detecting a newer version.
 
-## Documentation deployment
+## Deploying the documentation using the command line
 
-To deploy the documentation using the command line:
+As of now, you can deploy no more than 3 documentation versions at once.
```

Comment:
I'd change the meaning. The [issue](https://github.com/handsontable/handsontable/issues/8338) is about the memory limit that you have to set to be able to generate documentation (generate by building a docker image). 

The recommended limit is 8GB. Currently this would allow you to generate documentation that would contain 4 versions. Otherwise, the Fatal Error will be thrown.

![Kapture 2021-07-05 at 13 05 45](https://user-images.githubusercontent.com/571316/124462213-c3ce3680-dd91-11eb-82cb-f7c43e319e44.gif)